web/main/colors.css

```diff
@@ -6,6 +6,11 @@
   --fire1-color: oklch(70% 0.15 65);
   --fire2-color: oklch(58% 0.2 29);
   --error-color: var(--red-600);
+  --code-inline-bg-color: color-mix(in oklch, var(--neutral-700), var(--neutral-600));
+  --code-inline-color: oklch(100% 0 0);
+  --link-color-hover: oklch(from var(--link-color) l c h / 85%);
+  --link-color-visited: oklch(from var(--link-color) l c h / 75%);
```

Comment:
I'm afraid that 75% is not enough for good readability.

Also, the classic blue→violet color changes is a more standard way.

Reply:
Replaced with violet color from Harmony